ext/project_data.yaml

```diff
@@ -16,7 +16,7 @@ gem_default_executables: 'facter'
 gem_platform_dependencies:
   universal-darwin:
     gem_runtime_dependencies:
-      CFPropertyList: '~> 2.3.3'
+      CFPropertyList: '~> 2.2'
```

Comment:
Facter 2.0.1 shipped against CFPropertyList 2.2.6 ... so this will cover from that oldest version through to newest 2.3.5.

👍